libs/components/src/color-password/color-password.component.ts

```diff
@@ -0,0 +1,95 @@
+import { Component, HostBinding, Input } from "@angular/core";
+
+import { Utils } from "@bitwarden/common/misc/utils";
+
+type CharacterTypes = "letter" | "emoji" | "special" | "number";
+
+@Component({
+  selector: "color-password",
```

Comment:
issue: we start all of our CL components with `bit-`
```suggestion
  selector: "bit-color-password",
```